### PR TITLE
New Icon Set: `DesignSafe-Icons`

### PR DESCRIPTION
**Pull request by rstijerina**
## Overview: ##

- Add `DesignSafe-Icons` font files
- Add `DesignSafe-Icons.css` stylesheet
- Include new stylesheet in django base templates
- Update available icons in `AppListingEntry` model

## PR Status: ##

* [X] Ready.

## Testing Steps:

1. `./manage.py collectstatic`
2. TBD: Test icons in a CMS apps page

### CODE REVIEW

File: designsafe/apps/workspace/models/app_entries.py
Comment (wesleyboar) [inline designsafe/apps/workspace/models/app_entries.py:1]: 🙋‍♂️ If designer re-built/exported icons with lowercase names, would the migration still be necessary?

File: designsafe/apps/workspace/models/app_entries.py [hunk 1]
```diff
@@ -4,26 +4,35 @@
...
-    ("matlab", "MATLAB"),
-    ("ngl", "NGL"),
-    ("openfoam", "OpenFOAM"),
-    ("opensees", "OpenSees"),
-    ("paraview", "Paraview"),
-    ("qgis", "QGIS"),
-    ("rwhale", "rWHALE"),
-    ("stko", "STKO"),
-    ("swbatch", "swbatch"),
-    ("visit", "VisIt"),
```
Comment (wesleyboar) [inline designsafe/apps/workspace/models/app_entries.py:26]: 🧐 Is this being deleted? I don't see a new instance of it.

File: designsafe/apps/workspace/models/app_entries.py
Comment (wesleyboar) [reply designsafe/apps/workspace/models/app_entries.py:1]: Ah… hm. I think so, because new ones are added.

**Review by wesleyboar — COMMENTED** [review]
### Testing

1. ⚠️ **Works**… provided `static` is not fresh/empty.
    <sub>See https://github.com/DesignSafe-CI/portal/pull/1211.</sub>
2. <details><summary>⚠️ I added a Text plugin instance with a list of icon labels and <code>&lt;i&gt;</code> elements.</summary>

    ```html
    <ul>
		<li>Contract: <i class="icon icon-Contract"></i></li>
		<li>Expand: <i class="icon icon-Expand"></i></li>
		<li>Job-Status: <i class="icon icon-Job-Status"></i></li>
		<li>All-Hazards: <i class="icon icon-All-Hazards"></i></li>
		<li>New-Tab: <i class="icon icon-New-Tab"></i></li>
		<li>NGL-without-text: <i class="icon icon-NGL-without-text"></i></li>
		<li>Potree: <i class="icon icon-Potree"></i></li>
		<li>Generic-Vis: <i class="icon icon-Generic-Vis"></i></li>
		<li>HVSR: <i class="icon icon-HVSR"></i></li>
		<li>MPM: <i class="icon icon-MPM"></i></li>
		<li>Generic-App: <i class="icon icon-Generic-App"></i></li>
		<li>GiD: <i class="icon icon-GiD"></i></li>
		<li>Earth: <i class="icon icon-Earth"></i></li>
		<li>Water: <i class="icon icon-Water"></i></li>
		<li>Wind: <i class="icon icon-Wind"></i></li>
		<li>rWHALE: <i class="icon icon-rWHALE"></i></li>
		<li>Extract: <i class="icon icon-Extract"></i></li>
		<li>Hazmapper: <i class="icon icon-Hazmapper"></i></li>
		<li>Compress: <i class="icon icon-Compress"></i></li>
		<li>STKO: <i class="icon icon-STKO"></i></li>
		<li>OpenFOAM: <i class="icon icon-OpenFOAM"></i></li>
		<li>Blender: <i class="icon icon-Blender"></i></li>
		<li>MATLAB: <i class="icon icon-MATLAB"></i></li>
		<li>Paraview: <i class="icon icon-Paraview"></i></li>
		<li>Jupyter: <i class="icon icon-Jupyter"></i></li>
		<li>QGIS: <i class="icon icon-QGIS"></i></li>
		<li>OpenSees: <i class="icon icon-OpenSees"></i></li>
		<li>LS-DYNA: <i class="icon icon-LS-DYNA"></i></li>
		<li>Dakota: <i class="icon icon-Dakota"></i></li>
		<li>Clawpack: <i class="icon icon-Clawpack"></i></li>
		<li>Ansys: <i class="icon icon-Ansys"></i></li>
		<li>SWBatch: <i class="icon icon-SWBatch"></i></li>
    </ul>
    ```

    new DesignSafe icons

    </details>

File: designsafe/apps/workspace/models/app_entries.py (hunk 1 above)
Comment (wesleyboar) [reply designsafe/apps/workspace/models/app_entries.py:26]: Ah… Hm, I found it as Water, so never mind, perhaps.

File: designsafe/apps/workspace/models/app_entries.py
Comment (rstijerina) [reply designsafe/apps/workspace/models/app_entries.py:1]: The lowercase names could be added as a part of this PR FYI, I just elected to keep the casing that was in the supplied stylesheet to not interfere with existing class names. But seems like there is still a conflict, so I might need to add a prefix like `.ds-icon-`. If that were the case, would all lowercase class names be better?

File: designsafe/apps/workspace/models/app_entries.py (hunk 1 above)
Comment (rstijerina) [reply designsafe/apps/workspace/models/app_entries.py:26]: VisIt is going to become `Generic-Vis`

**Review by jarosenb — APPROVED** [review]

**Review by wesleyboar — APPROVED** [review]
Updated Test Result:

ds-icon-test

File: designsafe/apps/workspace/models/app_entries.py
Comment (wesleyboar) [reply designsafe/apps/workspace/models/app_entries.py:1]: Functionally, no. I originally proposed lowercase as a hasty guess at avoiding a migration.

<details><summary>I had left the comment unresolved to consider non-functional benefits to lowercase icon class name casing. I considered some, but they are minimal, and probably irrelevant if I understand correctly that this feature is a refactor of apps, not an augment.</summary>

* consistent with past development:
    * most updates are simple find/replaces (prefix `.ds-`)
    * less new names to learn
* no need for shift key when typing name

</details>

**Pull request merged**